pep-0584.rst

```diff
@@ -82,6 +82,29 @@ ChainMap will modify the original dict::
       {'eggs': 999}
 
 
+------------------
+``dict(d1, **d2)``
+------------------
+
+This "neat trick" is not well-known, and only works when ``d2`` is
+entirely string-keyed::
+
+    >>> d1 = {"spam": 1}
+    >>> d2 = {3665: 2}
+    >>> dict(d1, **d2)
+    Traceback (most recent call last):
+      ...
+    TypeError: keywords must be strings
+
+Even then, it should not be promoted as the "one obvious way".  Guido
+`said it best
+<https://mail.python.org/archives/list/python-dev@python.org/message/FM66WWN5YFCMRGMRWGHACOXLQYTWNXE2/>`_:
+
+    Apparently dict(x, \*\*y) is going around as "cool hack" for "call
+    x.update(y) and return x". Personally I find it more despicable
+    than cool.
```

Comment:
I don't think my emotional response adds much insight to the PEP (I also misstated the trick). I think the fact that it only works for string keys is enough to sink it.